package/MDAnalysis/coordinates/base.py

```diff
@@ -1330,6 +1330,95 @@ def __del__(self):
         self.close()
 
 
+class NewReader(ProtoReader):
```

Comment:
Is the plan to have this class completely replace `Reader`? If so, is it suppose to take over the `Reader` name? If not, having "new" in the name is not a smart move, at some point it will become the old one...